src/doc/rustdoc/src/the-doc-attribute.md

```diff
@@ -214,3 +214,31 @@ the `strip-hidden` pass is removed.
 Since primitive types are defined in the compiler, there's no place to attach documentation
 attributes. This attribute is used by the standard library to provide a way to generate
 documentation for primitive types.
+
+## `#[cfg(rustdoc)]`: Documenting platform-/feature-specific information
+
+Because of the way Rustdoc documents a crate, the documentation it creates is specific to the target
```

Comment:
How about this for rewriting these docs:

> For conditional compilation, Rustdoc treats your crate the same way the compiler does: Only things from the host target are available (or from the given `--target` if present), and everything else is "filtered out" from the crate. This can cause problems if your crate is providing different things on different targets, and you want your documentation to reflect all the available items you provide.
>
> If you want to make sure an item is seen by Rustdoc regardless of what platform it's being run from, you can apply `#[cfg(rustdoc)]` to it. Rustdoc sets this whenever it's building documentation, so anything that uses that flag will make it into documentation it generates. To apply this to an item with other `#[cfg]` filters on it, you can write something like `#[cfg(any(windows, rustdoc))]`. This will preserve the item either when built normally on Windows, or when being documented anywhere. For example:
>
> (current code sample)
>
> Here, the respective tokens can only be used by dependent crates on their respective platforms, but they will both appear in documentation.

Reply:
I really prefer your version so I'll just use it as is if you don't mind.